src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java

```diff
@@ -37,6 +37,19 @@ public static void main(String[] args) {
     static String getGreetingByHour(int hour) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        if (hour >= 6 && hour <= 11){
+            return "Good morning";
+        }
+        else if (hour >= 12 && hour <= 17){
+            return "Good day";
+        }
+        else if (hour >= 18 && hour <= 23){
+            return "Good evening";
+        }
+        else if (hour >= 0 && hour <= 5){
+            return "Good night";
+        } else {
+            return null;
+        }
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"